### PR TITLE
Added Office 365 one-click

### PR DESCRIPTION
**Pull request by zuccs**

### CODE REVIEW

**Comment by jacegu** [comment]
@zuccs thanks for the PR!

Could you please provide us with a link to the Office 365 documentation so we can validate that the records are the right ones? None of us in the team is familiar with the Office 365. Thanks in advance.

**Comment by jacegu** [comment]
Sorry @zuccs. Saw that you linked it in the email you sent us. Leaving it here for reference: https://support.office.com/en-au/article/Create-DNS-records-for-Office-365-at-any-DNS-hosting-provider-7b7b075d-79f9-4e37-8a9e-fb60c1d95166?ui=en-US&rs=en-AU&ad=AU#BKMK_verify

File: services/office-365/config.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      "content": "enterpriseenrollment.manage.microsoft.com",
+      "type": "CNAME",
+      "ttl": 3600
+    },
+    {
+      "content": "v=spf1 include:spf.protection.outlook.com -all",
+      "type": "SPF",
+      "ttl": 3600
+    },
+    {
```
Comment (aeden) [inline services/office-365/config.json]: Please resubmit without the record name and content broken out into separate fields.

For example:

```
{
  "name": "_sip._tls",
  "content": "1 443 sipdir.online.lync.com",
  ...
}
```

Comment (zuccs) [inline services/office-365/config.json]: Cool - have done that now @aeden .

**Comment by sbastn** [comment]
Thanks @zuccs for this submission! :+1:

**Pull request merged**